### PR TITLE
Initial commit of revoluzer tools

### PR DESCRIPTION
**Pull request by bernt-matthias**
The first two of some older tools which are apparently still used (revoluzer is more or less what I produced during my PhD a long time ago) :) Probably the first gene order analysis tools for Galaxy (with limited applicability .. probably only mitochondrial genomes).

University has shut down the old websites that provided these tools. So like for MITOS I would like to go Galaxy (unlike Galaxy revoluzer is C/C++, i.e. no other dependencies).

- [x] Requires https://github.com/bioconda/bioconda-recipes/pull/46403

FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [x] This PR adds a new tool or tool collection
* [ ] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

File: tools/revoluzer/macros.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<macros>
+    <token name="@TOOL_VERSION@">0.1.2</token>
```
Comment (bernt-matthias) [inline tools/revoluzer/macros.xml:2]:
```suggestion
    <token name="@TOOL_VERSION@">0.1.5</token>
```

File: tools/revoluzer/crex.xml [hunk 2]
```diff
@@ -0,0 +1,97 @@
+<tool id="crex" name="CREx" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
```
Comment (bgruening) [inline tools/revoluzer/crex.xml:1]:
```suggestion
<tool id="revoluzer_crex" name="CREx" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
```

File: tools/revoluzer/distmat.xml [hunk 3]
```diff
@@ -0,0 +1,119 @@
+<tool id="distmat" name="Compute distance matrix" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
```
Comment (bgruening) [inline tools/revoluzer/distmat.xml:1]:
```suggestion
<tool id="revoluzer_distmat" name="Compute distance matrix" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="20.01" license="MIT">
```

File: tools/revoluzer/crex.xml [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+            <when value="--crex1">
+                <param argument="--noalt" type="boolean" truevalue="" falsevalue="--noalt" checked="true" label="Compute alternatives for T+iT"/>
+                <param argument="--prinobp" type="boolean" truevalue="" falsevalue="--prinobp" checked="true" label="Compute breakpoint scenario for prime nodes"/>
+                <param argument="--primxalt" type="integer" min="1" value="2" label="maximal number of alternatives for prime nodes (I+TDRL, bp)"></param>
+            </when>
+            <when value="--bp"></when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data name="out" format="tabular"/>
```
Comment (bgruening) [inline tools/revoluzer/crex.xml:50]: add column names as metadata?

**Review by bgruening — APPROVED** [review]
Metadata for outputs or not you decide :-)

**Comment by bernt-matthias** [comment]
Found a bug.

**Comment by bernt-matthias** [comment]
Last commit should work with https://github.com/bioconda/bioconda-recipes/pull/46517

**Review by bgruening — APPROVED** [review]

**Pull request merged**